src/lang/Messages.properties

```diff
@@ -1876,12 +1876,20 @@ sites.spider.popup         = Spider...
 sites.showinsites.popup    = Show in Sites Tab
 
 spider.activeActionPrefix = Spidering: {0}
+spider.api.action.addDomainAlwaysInScope = Adds a new domain that's always in scope.
+spider.api.action.modifyDomainAlwaysInScope = Modifies a domain that's always in scope.
```

Comment:
That API endpoint just allows to modify one of the domains that is always in scope (like in the UI, change to regex, the value or enable/disable it).
Will improve the description of the endpoint and add describe the parameters.